### PR TITLE
Minutes for Packaging WG #14 15/11/2017

### PR DESCRIPTION
**Pull request by drbenmorgan**
This is mostly complete, just need to add the room booking and indico for the next meeting.

### CODE REVIEW

**Comment by drbenmorgan** [comment]
O.k. ready to go with review - room and link to next meeting added.

File: organization/_posts/2017-11-15-packaging.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "HSF Packaging Group Meeting #14, November 15, 2017"
+layout: default
+---
+
+# HSF Packaging Group Meeting #14, November 15, 2017
+
+#### *Present*: Ben Morgan, Chris Green, Dario Menasce, Graeme Stewart, Patrick Gartung, Martin Ritter, Emil Obreshkov, Javier, Rafal, Guilherme
```
Comment (graeme-a-stewart) [inline organization/_posts/2017-11-15-packaging.md]: Oh, I meant to fill in these names properly (I am bad with spelling surnames). Should be

Javier Cervantes Villanueva, Raphal Pacholek, Guilherme Amadio

**Comment by graeme-a-stewart** [comment]
Hi @drbenmorgan - thanks for these. Excepting the name change it looks good to me. The URL failures I will look into later when I do the HSF meeting minutes (see also https://github.com/HEP-SF/hep-sf.github.io/issues/184).

**Comment by drbenmorgan** [comment]
Thanks @graeme-a-stewart!

Re: #184 we've had exactly the same issue on SuperNEMO's Jekyll site, never resolved it, but always occurs on the Travis end, never locally...

**Pull request merged**